### PR TITLE
Feature Implementation: show warnings for one simulation only

### PR DESCRIPTION
**Pull request by ZDu-IDM**
PR addressed #71

### CODE REVIEW

File: examples/ben/example/run_1960-2004/run_simulations.py [hunk 1]
```diff
@@ -21,6 +20,20 @@ def _print_params():
     print("pull_from_serialization: ", params.pull_from_serialization)
 
 
+def _pre_run(experiment: Experiment, **kwargs):
+    """
+    Add extra work after run experiment.
```
Comment (stitova-idm) [inline examples/ben/example/run_1960-2004/run_simulations.py:25]: typo - "before running experiment", not "after"

File: examples/ben/example/run_1960-2004/run_simulations.py [hunk 2]
```diff
@@ -70,17 +83,20 @@ def run_experiment(show_warnings: bool = True, **kwargs):
...
-    experiment.run(wait_until_done=False, wait_on_done=False)
+    _pre_run(experiment, **kwargs)
+    experiment.run(wait_until_done=False, wait_on_done=False, **kwargs)
     _post_run(experiment, **kwargs)
 
 
 if __name__ == "__main__":
+    """
+    - show_warnings_once=True:  show api warnings for only one simulation
+    - show_warnings_once=False: show api warnings for all simulations
```
Comment (stitova-idm) [inline examples/ben/example/run_1960-2004/run_simulations.py:97]: Thumbs up

File: examples/monique/calibration/seasonality_calibration/02_seasonality_calibration/run_calibration.py [hunk 3]
```diff
@@ -28,6 +27,20 @@ def print_params():
     print("burnin_ids: ", params.burnin_ids)
 
 
+def _pre_run(**kwargs):
+    """
+    Add extra work after run experiment.
```
Comment (stitova-idm) [inline examples/monique/calibration/seasonality_calibration/02_seasonality_calibration/run_calibration.py:32]: "before" not "after" typo

File: examples/monique/calibration/seasonality_calibration/02_seasonality_calibration/manifest.py [hunk 4]
```diff
@@ -9,13 +9,16 @@
...
 # specify user data for testing
-# USER_PATH = r'C:\Projects\emodpy-snt\data'
+USER_PATH = r'C:\Projects\emodpy-snt\data'
 
 # load project path
 data_path, project_path = load_box_paths(user_path=USER_PATH, country_name='Example')
 input_dir = os.path.join(project_path, "simulation_inputs")
 
 sif_id = None
+sif_id = r'C:\Work\images\dtk_run_rocky_py39.sif'
```
Comment (stitova-idm) [inline examples/monique/calibration/seasonality_calibration/02_seasonality_calibration/manifest.py:24]: I'll be changing this to a relative path to the sif I add, so this is ok.

File: examples/monique/calibration/seasonality_calibration/02_seasonality_calibration/manifest.py [hunk 5]
```diff
@@ -9,13 +9,16 @@
 schema_file = os.path.join(download_dir, "schema.json")
 eradication_path = os.path.join(download_dir, "Eradication")
 
+plugins_folder = os.path.join(download_dir, "plugins")
```
Comment (stitova-idm) [inline examples/monique/calibration/seasonality_calibration/02_seasonality_calibration/manifest.py:12]: plugins back?

File: examples/monique/calibration/seasonality_calibration/02_seasonality_calibration/run_calibration.py [hunk 6]
```diff
@@ -182,5 +198,5 @@ def run_calibration(directory: str = '.', show_warnings: bool = True, **kwargs):
     # print("...done.")
 
     # Specify local folder for calibration results
-    directory = r'C:\Projects\emodpy-snt\data\TEST_DEST_CALIBRA4'
-    run_calibration(directory=directory, show_warnings=False)
+    directory = r'C:\Projects\emodpy-snt\data\TEST_DEST_CALIBRA5'
```
Comment (stitova-idm) [inline examples/monique/calibration/seasonality_calibration/02_seasonality_calibration/run_calibration.py:201]: should directory for calibration results be in the manifest?

File: examples/monique/run_future_scenarios/run_simulations.py [hunk 7]
```diff
@@ -18,6 +17,20 @@ def _print_params():
     print("burnin_id: ", params.burnin_id)
 
 
+def _pre_run(experiment: Experiment, **kwargs):
+    """
+    Add extra work after run experiment.
```
Comment (stitova-idm) [inline examples/monique/run_future_scenarios/run_simulations.py:22]: "before"

**Review by stitova-idm — APPROVED** [review]

File: examples/monique/calibration/seasonality_calibration/02_seasonality_calibration/manifest.py (hunk 5 above)
Comment (ZDu-IDM) [reply examples/monique/calibration/seasonality_calibration/02_seasonality_calibration/manifest.py]: Fixed

File: examples/monique/calibration/seasonality_calibration/02_seasonality_calibration/run_calibration.py (hunk 6 above)
Comment (ZDu-IDM) [reply examples/monique/calibration/seasonality_calibration/02_seasonality_calibration/run_calibration.py]: Good suggestion! Now added directory to manifest.py

File: examples/monique/run_future_scenarios/run_simulations.py (hunk 7 above)
Comment (ZDu-IDM) [reply examples/monique/run_future_scenarios/run_simulations.py:22]: Fixed all such typo, thanks!

**Pull request merged**